modules/setting/indexer.go

```diff
@@ -58,7 +58,7 @@ func loadIndexerFrom(rootCfg ConfigProvider) {
 		if !filepath.IsAbs(Indexer.IssuePath) {
 			Indexer.IssuePath = filepath.ToSlash(filepath.Join(AppWorkPath, Indexer.IssuePath))
 		}
-		fatalDuplicatedPath("issue_indexer", Indexer.IssuePath)
+		checkOverlappedPath("indexer.ISSUE_INDEXER_PATH", Indexer.IssuePath)
```

Comment:
I think it's better if we notify the users explicitly what the section is:
```suggestion
		checkOverlappedPath("[indexer].ISSUE_INDEXER_PATH", Indexer.IssuePath)
```